.github/workflows/go-update-commenter.yml

```diff
@@ -0,0 +1,65 @@
+name: "Go update commenter"
+
+on:
+  pull_request:
+    # Only run on PR label events (in particular not on every commit)
+    types: [ labeled ]
+
+jobs:
+  old-versions-match:
+    # Only run if the PR is labeled with 'go-update'
+    if: ${{ github.event.label.name == 'go-update' }}
+    runs-on: ubuntu-latest
+    steps:
+      # get the Go version of the target branch
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.base_ref }}
+      - name: Get former Go version
+        id: former_go_version
+        run: |
+          echo version="$(cat .go-version)" >> $GITHUB_OUTPUT
+
+      # get the Go version of the PR branch
+      - uses: actions/checkout@v3
+      - name: Get current Go version
+        id: new_go_version
+        run: |
+          echo version="$(cat .go-version)" >> $GITHUB_OUTPUT
+
+      # build the comment
+      - name: Build full comment
+        id: old_versions
+        run: |
+          set -euxo pipefail
+          # build the base of the Github URL to the current commit
+          GITHUB_HEAD_URL='${{ github.server_url }}/${{ github.repository }}/blob/${{ github.sha }}'
+          {
+            echo "matches<<EOF"
```

Comment:
Wouldn't it be simpler/more readable to output all commands in `$GITHUB_OUTPUT` directly without the `{ cat <<< EOF ... EOF } >> ` machinery?

Reply:
For the logic around the format (`echo "matches<<EOF"` ... `echo EOF`) this is the github doc for how to get multiline output from a workflow step https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

Reply:
If that's the recommended way then fine by me, I find it a bit less readable but to everyone their own tastes :)

FTR I meant "adding `>> $GITHUB_OUTPUT`"